README.md

```diff
@@ -178,8 +179,7 @@ mod flipper {
     }
 }
 ```
-
-The [`flipper/src/lib.rs`](https://github.com/paritytech/ink-examples/blob/v4.0.0/flipper/lib.rs)
+The [`flipper/src/lib.rs`](https://github.com/paritytech/ink-examples/blob/v4.0.1/flipper/lib.rs)
```

Comment:
Is there the need to create a new tag for this in the `ink-examples`? The `4.0.0` examples should still be compatible, otherwise it wouldn't be able to be a patch release in the first place. 

But perhaps it is just easier to create tag per release regardless.

Also we need to add a step to the release checklist for creating this branch where required.

Reply:
> Is there the need to create a new tag for this in the `ink-examples`? The `4.0.0` examples should still be compatible, otherwise it wouldn't be able to be a patch release in the first place.
> 
> But perhaps it is just easier to create tag per release regardless.
> 
> Also we need to add a step to the release checklist for creating this branch where required.

I reckon there is nothing wrong to indicate the latest version in examples.